src/pages/explore/ui/pair-card.tsx

```diff
@@ -58,8 +62,15 @@ export type PairCardProps =
     };
 
 export const PairCard = ({ loading, summary }: PairCardProps) => {
+  // TODO: change to correct dates
+  const today = subDays(new Date(), 3);
+  const yesterday = subDays(new Date(), 4);
```

Comment:
Needs changes. I put this info because we have candles from 4 days ago right now but the data is completely missing from pindexer for this weekend